services/scabbard/libscabbard/src/store/scabbard_store/diesel/operations/list_ready_services.rs

```diff
@@ -108,6 +111,22 @@ where
                     })?,
             );
 
+            ready_services.append(
```

Comment:
should this query also check to make sure that the services with unexecuted supervisor notifications are in the finalized state?

Reply:
added